2_form.html

```diff
@@ -0,0 +1,72 @@
+<h1>Payment Form</h1>
+<h4>Requierd Fields are followed by .*.</h4>
+<p>Contact Information</p>
+<div style="line-height: 1cm;">
+   <form action="" method="GET">
+      <fieldset>
+         <legend>Contact Information</legend>
+         <fieldset>
+            <legend>Choose</legend>
+            <div>
+               <label>Are you Teacher or Student ?</label>
+            </div>
+            <div>
+               <label for="type">Student</label>
+               <input name="Student or Teacher" type="radio" value="Student" >
```

Comment:
Student and teacher radio sre not connected you can have both selected. Name attr missing